source/asm/TOOLCHAIN_ARM_STD/CortexContextSwitch.S

```diff
@@ -0,0 +1,293 @@
+; The MIT License (MIT)
```

Comment:
Is this file meant to replace https://github.com/lancaster-university/microbit-dal/blob/master/source/asm/CortexContextSwitch.s.armcc ? If not, can `CortexContextSwitch.s.armcc` be moved to a subdirectory similar to this new file path pattern?

Reply:
The makefile requires that file to exist there so that it can select which file to copy to `asm/CortexContextSwitch.s` at compile time.